eagle-core/eagle-alert-parent/eagle-alert/alert-engine/src/main/java/org/apache/eagle/alert/engine/spout/SpoutOutputCollectorWrapper.java

```diff
@@ -118,6 +118,8 @@ public SpoutOutputCollectorWrapper(CorrelationSpout spout,
         }
 
         StreamEvent event = convertToStreamEventByStreamDefinition((Long) convertedTuple.get(2), m, sds.get(streamId));
+        LOG.info("Spout from topic {} emit event: {}", topic, event);
```

Comment:
When the throughput increases, this log (printing log per event) may be the bottle neck.